proposals/statics-in-interfaces.md

```diff
@@ -56,20 +56,59 @@ Static interface members today are implicitly non-virtual, and do not allow `abs
 
 ### Proposal
 
-#### Abstract virtual members
-Static interface members other than fields are allowed to also have the `abstract` modifier. Abstract static members are not allowed to have a body (or in the case of properties, the accessors are not allowed to have a body). 
+#### Static virtual members
+Static interface members other than fields are allowed to have an `abstract` or `virtual` modifier.
+
+Static virtual members declared `abstract` are not allowed to have a body (or in the case of properties, the accessors are not allowed to have a body). 
+
+Static virtual members declared `virtual` must provide a default implementation in the form of a member body (or in the case of properties, accessor bodies).
 
 ``` c#
-interface I<T> where T : I<T>
+interface I1<T> where T : I1<T>
 {
-    static abstract void M();
-    static abstract T P { get; set; }
-    static abstract event Action E;
+    static abstract void M1();
+    static virtual void M2() => Console.WriteLine("Default behavior");
+    
+    static abstract T P1 { get; set; }
+    static virtual T P2 { get => T.P1; set => T.P1 = value; }
+    
+    static abstract event Action E1;
+    static virtual event Action E2 { add => T.E1 += value; remove => T.E1 -= value; }
+    
     static abstract T operator +(T l, T r);
+    static virtual T operator -(T l, T r) => l;
+    static abstract bool operator ==(T l, T r);
```

Comment:
At some point (not necessarily in his PR) it would be good to specify changes for requirements around signatures of operators, if any.

Reply:
That is if "Operator restrictions" section is insufficient.

Reply:
I agree we should take another look at this.